medmodels/data_synthesis/mtgan/mtgan.pyi

```diff
@@ -0,0 +1,73 @@
+from pathlib import Path
+from typing import Dict, Optional, Type
+
+from torch import nn
+
+from medmodels.data_synthesis.mtgan.builder import MTGANBuilder
+from medmodels.data_synthesis.mtgan.model.gan import (
+    TrainingHyperparameters,
+    TrainingHyperparametersOptional,
+)
+from medmodels.data_synthesis.mtgan.modules.postprocessor import (
+    AttributeType,
+    PostprocessingHyperparameters,
+    PostprocessingHyperparametersOptional,
+)
+from medmodels.data_synthesis.mtgan.modules.preprocessor import (
+    PreprocessingHyperparameters,
+    PreprocessingHyperparametersOptional,
+)
+from medmodels.data_synthesis.mtgan.mtgan_model import MTGANModel
```

Comment:
This could maybe cause problems with a circular input? We're using MTGANModel in MTGAN and MTGAN in MTGANModel.

Reply:
we will need both :( We will need to pay attention to circular imports. Already done in full version of MTGAN.